src/main/resources/db/migration/V2023.10.25.08.53.44__create_doc_type_label_for_user_files.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE user_files
+    ADD COLUMN doc_type_label VARCHAR(255) DEFAULT '${user_file_doc_type_default_label}';
```

Comment:
Should we add a test to ensure the default is set when used?

Reply:
I didn't address this one, but can look at it.

Reply:
I tried to add a test for this but am seeing the test db act differently then the live db. Namely defaults and required fields are not getting set in test. This makes it so I cannot test the default value of the doc type field. I did create tests and check it in, but it's disabled because the db isn't correct.